lib/Makefile

```diff
@@ -259,7 +259,7 @@ blas_magma.o: blas_magma.cpp $(HDRS)
 	$(CXX) $(MAGMA_INCLUDE) $(CXXFLAGS) $(MAGMA_FLAGS) $< -c -o $@
 
 blas_magma_gpu.o: blas_magma.cu $(HDRS)
-	$(NVCC) $(NVCCFLAGS) $(MAXREG) $< -c -o $@
+	$(NVCC) -DMAGMA_LIB $(NVCCFLAGS) $(MAXREG) $< -c -o $@
```

Comment:
What is the point of always defining `MAGMA_LIB` here?
If it is always defined anyway, can't you just do that in the source file?

Or should that be a configure option?